lang/java/reef-common/src/main/java/org/apache/reef/driver/ContextAndTaskSubmittable.java

```diff
@@ -60,5 +60,4 @@
   void submitContextAndServiceAndTask(final Configuration contextConfiguration,
                                       final Configuration serviceConfiguration,
                                       final Configuration taskConfiguration);
-
```

Comment:
I'll fix it during the merge.

Background: I brought it up because this conflates a functionality change with a formatting change. In this case, this is even the only change in this file.

This can get in the way of understanding the history of the project and can hamper our ability to debug issues: If there is a bug involving this file in the future and we ask which commit changed it last, this one will come up. And we might not notice that it was a non-functionality change and spend debugging time on it.